src/app/pages/SetTimer/SetTimer.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import Button from '../../components/Button/Button';
+
+export default function SetTimer(): JSX.Element {
+  const lastDuration = JSON.parse(localStorage.getItem('Duration') || '[]');
+  const [duration, setDuration] = useState(lastDuration);
+
+  function saveDuration() {
+    localStorage.setItem('Duration', JSON.stringify(duration));
+    console.log(duration);
+  }
+
+  return (
+    <>
+      <PageContainer>
+        <Header>Set up the Timer</Header>
+        <TimeCircle>
+          <Duration>{duration} Min</Duration>
+        </TimeCircle>
+        <Range
+          type="range"
+          min="0"
+          max="60"
+          value={duration}
+          onChange={(event) => setDuration(parseInt(event.target.value))}
+        ></Range>
+        <Link to="/timer">
+          <Button saveTime={saveDuration}>Start</Button>
+        </Link>
+      </PageContainer>
+    </>
+  );
+}
+
+const PageContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 2rem;
+  height: 100vh;
+  width: 100vw;
+  background: var(--white);
+  background-size: cover;
+  max-width: 400px;
```

Comment:
Why not using rem here as well?
You have used px above as well in older code for max-width, if I remember right.